darts-api-stub/authentication/external.js

```diff
@@ -38,7 +38,7 @@ router.get('/logout', (_, res) => {
 });
 
 router.get('/login', (_, res) => {
-  res.render('external-login');
+  res.render('external-login', { callbackUrl: 'http://localhost:3000/auth/internal/callback' });
```

Comment:
Shouldn't this be the external callback?

Reply:
🤦